### PR TITLE
DEVDOCS-2332: Update Resources section of API Docs articles

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2332](https://jira.bigcommerce.com/browse/DEVDOCS-2332)

## What changed?
* updated Resources section of API Docs article to match the new order (Articles, Endpoints, Webhooks)

### CODE REVIEW

**Review by traci707 — COMMENTED** [review]
Everything looks good in Updated related resources; added OAuth scope parameters. I am just curious if you meant to exclude Customer Groups and Password Validation from the list of Endpoints.

File: docs/api-docs/storefront/graphql/graphql-api-overview.md [hunk 1]
```diff
@@ -29,7 +28,7 @@ BigCommerce's GraphQL Storefront API makes it possible to query storefront data
 
 Additionally, by leveraging the power of [GraphQL](https://graphql.org/), data for multiple resources can be returned from a single API call, which simplifies integration and increases performance so that developers can focus on building delightful shopper experiences.
 
-This article is a general overview of the capabilities and usage of BigCommerce's GraphQL Storefront API; it includes sections on authentication and how to access a store's GraphQL Playground. To see specific examples of how GraphQL can be used to query storefront data, see [GraphQL Storefront API Code Samples](https://developer.bigcommerce.com/api-docs/storefront/graphql/graphql-storefront-api-samples).
+This article is a general overview of the capabilities and usage of BigCommerce's GraphQL Storefront API; it includes sections on authentication and how to access a store's GraphQL Playground. To see specific examples of how GraphQL can be used to query storefront data, see [GraphQL Storefront API Example Queries](https://developer.bigcommerce.com/api-docs/storefront/graphql/graphql-storefront-api-samples).
```
Comment (traci707) [inline docs/api-docs/storefront/graphql/graphql-api-overview.md:31]: (Optional Suggestion) You can consider removing unnecessary words to be more concise, for example:
This article is a general overview of BigCommerce's GraphQL Storefront API; it includes sections on authentication and how to access a store's GraphQL Playground. To see specific examples of how GraphQL can be used to query storefront data, see [GraphQL Storefront API Example Queries](https://developer.bigcommerce.com/api-docs/storefront/graphql/graphql-storefront-api-samples).

**Review by traci707 — CHANGES_REQUESTED** [review]
In GraphQL Storefront API Overview, capitalize "the" in the sentence below:

The next thirty could be retrieved by making a new query and passing in the endCursor from the first page of results:

File: docs/api-docs/payments/payments-api-overview.md [hunk 2]
```diff
@@ -629,21 +629,21 @@ The Payment Processing API is for processing payments through a store's payment
 **Are offline payment methods supported?**
 The Payments API is designed to process credit card payments through supported payment gateways; it does not expose methods for processing [offline payment methods](https://support.bigcommerce.com/s/article/Offline-Payment-Methods) such as cash on delivery.
```
Comment (traci707) [inline docs/api-docs/payments/payments-api-overview.md:631]: The Payments API processes credit card payments through supported payment gateways; it does not expose methods for processing [offline payment methods](https://support.bigcommerce.com/s/article/Offline-Payment-Methods) such as cash on delivery.

File: docs/api-docs/storefront/scripts-overview.md [hunk 3]
```diff
@@ -147,7 +147,6 @@ Scripts can not be injected to:
 - giftcertificates.php
 - sitemap.php
 
-
 ## Related resources
```
Comment (traci707) [inline docs/api-docs/storefront/scripts-overview.md:150]: Did you add Related resources to "On this page"?

**Review by traci707 — APPROVED** [review]

**Review by uwikaiddi — CHANGES_REQUESTED** [review]
See @traci707 comments

**Comment by uwikaiddi** [comment]
@traci707, please be careful not to approve if you also requested changes tot he PR.

**Comment by traci707** [comment]
@uwikaiddi you are right.  Some files didn't need changes but some did.

File: docs/api-docs/developers-guide-headless.md [hunk 4]
```diff
@@ -233,14 +233,23 @@ Merchants can use BigCommerce's [PCI DSS AOC](https://support.mybigcommerce.com/
...
-### Tools
-- [Checkout SDK](https://github.com/bigcommerce/checkout-sdk-js)
-- [WordPress Plugin](https://wordpress.org/plugins/bigcommerce/) (WordPress Plugin Directory)
-
-### Related endpoints
+## Related resources
 
+### Articles
+- [Customers Overview](https://developer.bigcommerce.com/api-docs/customers/customers-subscribers-overview)
+- [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api)
```
[the remark this answers is not in the document]
Comment (ektravel) [reply docs/api-docs/developers-guide-headless.md:250]: In this case, it is an overview article.

File: docs/api-docs/storefront/scripts-overview.md (hunk 3 above)
Comment (ektravel) [reply docs/api-docs/storefront/scripts-overview.md:150]: Yes

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**